autoscaler/controllers/nodecollector/manager.go

```diff
@@ -25,10 +25,12 @@ func SetupWithManager(mgr ctrl.Manager, imagePullSecrets []string, odigosVersion
 				predicate.And(&odigospredicate.OdigosCollectorsGroupClusterPredicate, &odigospredicate.ReceiverSignalsChangedPredicate{}),
 			)).
 		Complete(&CollectorsGroupReconciler{
-			Client:           mgr.GetClient(),
-			Scheme:           mgr.GetScheme(),
-			ImagePullSecrets: imagePullSecrets,
-			OdigosVersion:    odigosVersion,
+			nodeCollectorBaseReconciler: nodeCollectorBaseReconciler{
```

Comment:
nit: now that it's all the same, we can create it just once and use the same instance for each controller in this file